packages/console/src/components/ImagePlaceholder/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import defaultPlaceholder from '@/assets/images/default-placeholder.svg';
+
+import * as styles from './index.module.scss';
+
+const ImagePlaceholder = () => {
+  return <img src={defaultPlaceholder} className={styles.placeholder} />;
```

Comment:
Better have `className` as a pass-in parameter, so the caller of this component can decide the size of it?

Reply:
would prefer fixed first, in the future just like `<Button>`, have some preset sizes